clang/lib/Tooling/DependencyScanning/ModuleDepCollector.cpp

```diff
@@ -482,16 +482,9 @@ ModuleDepCollectorPP::handleTopLevelModule(const Module *M) {
   MD.ID.ModuleName = M->getFullModuleName();
   MD.IsSystem = M->IsSystem;
 
-  ModuleMap &ModMapInfo =
-      MDC.ScanInstance.getPreprocessor().getHeaderSearchInfo().getModuleMap();
-
-  OptionalFileEntryRef ModuleMap = ModMapInfo.getModuleMapFileForUniquing(M);
-
-  if (ModuleMap) {
-    SmallString<128> Path = ModuleMap->getNameAsRequested();
-    ModMapInfo.canonicalizeModuleMapPath(Path);
-    MD.ClangModuleMapFile = std::string(Path);
-  }
+  assert(!M->PresumedModuleMapFile.empty() &&
```

Comment:
There's a comment in Module.h: `/// Only non-empty when building from preprocessed source.` that should probably be deleted.